website/content/docs/secrets/kmip-profiles.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 layout: docs
-page_title: KMIP - Profiles Support
-description: |-
+page_title: KMIP profiles version 1.4
```

Comment:
I don't think this change is accurate.  @victorr ?

Reply:
I agree with Scott, I think the original title is better.

Reply:
Since this is a hidden file, I can revert it back. 

Ideally, we want the `page_title` and the H1 header to match. Do you want to keep the H1 header as-is though? 

Right now, the H1 header is: `# KMIP profiles version 1.4`

Reply:
I think it is fine to change the H1 header to match the title. The second paragraph informs of the KMIP version, so it is fine for the header to drop it.